test/README.md

```diff
@@ -115,6 +115,8 @@ permissions inside the TaskRun to run the Kaniko e2e test and GCS taskrun test.
   `KANIKO_SECRET_CONFIG_FILE` is used to generate Kubernetes secret to access
   GCS bucket. This e2e test requires valid service account configuration json
   but it does not require any role binding.
+- In Storage artifact bucket, GCP service account JSON key file at path
+  `KANIKO_SECRET_CONFIG_FILE` is used to create/delete a bucket.
```

Comment:
I think we need to refactor/reword this paragraph. It might not be in the scope of this PR. We can leave TODO here. We need to break down  "GCP serviceaccount JSON key file" . One suggestion would be the following

In the Storage artifact bucket test, the JSON key file for the GCP service account stored at path `KANIKO_SECRET_CONFIG_FILE ` is used to create/delete a GCS bucket.

Reply:
now that we're using `KANIKO_SECRET_CONFIG_FILE` for the bucket test as well, would it make sense to rename `KANIKO_SECRET_CONFIG_FILE` to something more general?

Reply:
renamed to `GCP_SERVICE_ACCOUNT_KEY_PATH` and reworded the readme